gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -67,7 +63,7 @@ The following minimum requirements apply to a small Kubernetes cluster used for
 
 * Support for the creation of a `Service` resource of the `LoadBalancer` type.
 +
```

Comment:
@seblaz ok, here's the suggested update, below. Pls check it out when you get a chance.